src/components/space/charts/ActionsByDayChart.tsx

```diff
@@ -62,21 +62,31 @@ const ActionsByDayChart = (): JSX.Element | null => {
   }
 
   const title = t('ACTIONS_BY_DAY_TITLE');
-  if (!aggregateData?.size) {
+  if (!aggregateData?.length) {
     return <EmptyChart chartTitle={title} />;
   }
 
-  const formattedAggregateData = (
-    aggregateData.toArray() as { aggregateResult: number; createdDay: Date }[]
-  )
-    // sort by creation date
-    .sort((a, b) => a.createdDay.getTime() - b.createdDay.getTime())
-    .map((d) => ({
-      averageCount: d.aggregateResult,
-      date: `${d.createdDay.getDate()}-${
-        d.createdDay.getMonth() + 1
-      }-${d.createdDay.getFullYear()}`,
-    }));
+  // sort by creation date
+  aggregateData.sort((a, b) => {
```

Comment:
If we want to sort in a non-mutable way:

```suggestion
  const aggregateDataSorted = [...aggregateData].sort((a, b) => {
```